README.md

```diff
@@ -1,3 +1,47 @@
-# JavaMail S/MIME Transport
+# Java Mail S/MIME Transport
+
+When this library is added to a project you may configure a Java Mail session to use the `smtp+smime` or `smtps+smime` protocol to have the transport sign outgoing mail with a mail certificate stored in a Java keystore (JKS) file.
+
+## Configuration
+| Property                                | Description |
+|-----------------------------------------|-------------|
+| mail.transport.protocol                 | Special protocol to use: `smtp+smime` or `smtps+smime` |
+| mail.keystore.file                      | Keystore file containing certificate |
+| mail.keystore.password                  | Password for keystore file |
+| mail.keystore.`email address`.password  | Password for `email address` alias in keystore
+
+If your keystore only contains one email alias and its password is the same as the keystore's, you can omit the mail.keystore.`email`.address configuration.
```

Comment:
The "escaped" email address part should probably be handled the same way here as it is in the table above

Reply:
Thanks. I'm making several changes that I'll push here in a minute.